pyomeca/frame_dependent.py

```diff
@@ -494,7 +494,9 @@ def _parse_c3d(c3d, prefix):
         raise NotImplementedError("_parse_c3d_info is an abstract function")
 
     @classmethod
-    def from_c3d(cls, filename, idx=None, names=None, prefix=None):
+    def from_c3d(
```

Comment:
I think you included an old PR that we refused

Reply:
This is a monster that will always strike back!